main.go

```diff
@@ -154,6 +153,14 @@ func runCopy(db *redshift.Redshift, inputConf s3filepath.S3File, inputTable reds
 	if err := tx.Commit(); err != nil {
 		return fmt.Errorf("err committing transaction: %s", err)
 	}
+
+	if truncate {
+		// If we've truncated the table we should run vacuum and analyze to sort the
+		// data, remove the deleted rows, and update the statistics.
+		if err := db.VacuumAnalyze(inputConf.Schema, inputTable.Name); err != nil {
```

Comment:
Do you think it's worth limiting this to just a `VACUUM DELETE ONLY`, for efficiency http://docs.aws.amazon.com/redshift/latest/dg/r_VACUUM_command.html ?

We could then run a full analyze later on.